.eslintrc.json

```diff
@@ -6,6 +6,7 @@
     "rules": {
         "max-len": "off",
         "brace-style": ["error", "stroustrup"],
-        "no-unused-vars": ["error", { "args": "none" }]
+        "no-unused-vars": ["error", { "args": "none" }],
+	"no-underscore-dangle": ["error", { "allowAfterThis": true }]
```

Comment:
By [default](https://github.com/airbnb/javascript#naming--leading-underscore) it is set to not allow underscores because it implies that a method or property is private when in fact it is really still public.  Allowing only after "this" at least to the linter enforces that those methods are only used internally to the class.